UnitsNet/CustomCode/Wrappers/ReferencePressure.cs

```diff
@@ -25,6 +25,21 @@ namespace UnitsNet.CustomCode.Wrappers
     /// </summary>
     public struct ReferencePressure
     {
+        /// <summary>
+        ///     Represents the pressure at which _pressure is referenced (1 atm default)
+        /// </summary>
+        public static Pressure AtmosphericPressure { get; set; }
```

Comment:
I don't think this is supposed to be static?

Reply:
I would agree.